views/membership.py

```diff
@@ -53,8 +57,13 @@ def create(self, request, *args, **kwargs):
         try:
             group = Group.objects.get(pk=group)
             if not has_admin_rights(person, group):
+                logger.warning(
+                    f'Person trying to create Membership does'
+                    f' not have admin rights for that group'
+                )
                 raise PermissionDenied
         except Group.DoesNotExist:
+            logger.warning(f'Group does not exist with pk={group}')
```

Comment:
Exceptions should preferably write into `logger.error` but a proper resolution would be a formalised set of rules to determine what to log at what level.